### PR TITLE
Fix #2738: participant scrollbar

### LINKED ISSUE #2738: Fix scroll bars for participants
Image

- Disable overflow if scrolling is not necessary
- Style scrollbars according to other elements (e.g. columns)

### PR DESCRIPTION
**Pull request by Benjosh95**
## Description

Fixed the styling and behaviour of the participant-list scrollbar. 
The scrollbar only appears when enough users have joined the board.
Otherwise the participants take the same with as the other elements in its settingpage.  

## Changelog

Added a check for an overflow of the participant list and changed styling and scrolling behaviour accordingly.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

## (Optional) Visual Changes

Before:

image

Changes:

image

### CODE REVIEW

File: src/components/SettingsDialog/Participants/Participants.tsx [hunk 1]
```diff
@@ -18,12 +18,27 @@ export const Participants = () => {
...
   const self = useAppSelector((state) => state.participants!.self);
   const participants = useAppSelector((state) => [state.participants!.self, ...(state.participants?.others ?? [])]);
   const existsAtLeastOneReadyUser = participants.some((p) => p.ready);
 
+  useEffect(() => {
+    const scrollableDiv = scrollRef.current;
+    if (!scrollableDiv) {
+      return undefined;
+    }
+    const mutationObserver = new MutationObserver(() => {
```
Comment (Resaki1) [inline src/components/SettingsDialog/Participants/Participants.tsx:34]: Do we really need a useEffect and Observer for this? Can't we just use `overflow-x: hidden` and `overflow-y: scroll`?

**Review by Resaki1 — CHANGES_REQUESTED** [review]
The participant search input changes height when the scrollbar appears, which looks weird

**Comment by Benjosh95** [comment]
> The participant search input changes height when the scrollbar appears, which looks weird

Fixed

File: src/components/SettingsDialog/Participants/Participants.tsx (hunk 1 above)
Comment (Benjosh95) [reply src/components/SettingsDialog/Participants/Participants.tsx:34]: blub

**Review by Resaki1 — APPROVED** [review]

**Pull request merged**